pymc/distributions/discrete.py

```diff
@@ -117,6 +117,12 @@ def dist(cls, n, p, *args, **kwargs):
         # mode = at.cast(tround(n * p), self.dtype)
```

Comment:
```suggestion
```

Same here, we should remove the old commented out code, and use `at.round` in the new `get_moment`

Reply:
Is the `at.cast` also necessary here?

Reply:
Not anymore. That's done automatically by the function that calls get_moment